### PR TITLE
Update Kolibri installer license file.

### PR DESCRIPTION
**Pull request by mrpau-richard**
### Summary

- I updated the Kolibri windows installer license file. 
- I removed old kalite codes at Kolibri installer source.

Hi @radinamatic can you test this [Kolibri windows installer](https://drive.google.com/open?id=0B5Tyi-tRKOFMUmpzZEQ0aC1KMTg) with the updated license file. 

/cc @cpauya  @radinamatic

### CODE REVIEW

File: windows/kolibri/LICENSE [hunk 1]
```diff
@@ -1,20 +1,21 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) 2015 Learning Equality
+Copyright (c) 2016 Learning Equality
```
Comment (radinamatic) [inline windows/kolibri/LICENSE]: Are we supposed to keep the 2016 here for some reason? Or go directly to 2017... 😛 
@jamalex @indirectlylit

Comment (mrpau-richard) [inline windows/kolibri/LICENSE]: Hi @radinamatic I updated the LICENSE file year to 2017.

Comment (indirectlylit) [inline windows/kolibri/LICENSE]: should this file be updated also?

**Comment by indirectlylit** [comment]
should we get this PR merged in?

there was still one other '2016':
https://github.com/learningequality/kolibri-installer-windows/pull/1/files#diff-1d562789e442b63cef9c7517d5a86403R3

**Comment by radinamatic** [comment]
I'd say we should, but this might be stale... 

@mrpau-richard can you merge these 2 license year updates?

**Comment by mrpau-richard** [comment]
@radinamatic I already updated the two license file. Merging this.

**Pull request merged**